android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/CustomViewHolder.kt

```diff
@@ -47,4 +47,9 @@ sealed class CustomViewHolder(view: View) : RecyclerView.ViewHolder(view) {
         val addButton: ImageView,
         val allowedInPrivateBrowsingLabel: ImageView,
     ) : CustomViewHolder(view)
+
+    /**
+     * A view holder for displaying a "find more add-ons" button.
+     */
```

Comment:
As this could be used for other consumers, maybe we could generic here.
Maybe "A view holder for displaying an item at the end of add-ons list", what do you think?